backend/internal/system/jwt/service.go

```diff
@@ -55,6 +55,8 @@ type JWTServiceInterface interface {
 	Init() error
 	GetPublicKey() *rsa.PublicKey
 	GenerateJWT(sub, aud, iss string, validityPeriod int64, claims map[string]interface{}) (string, int64, error)
+	VerifyJWT(jwtToken string, jwtPublicKey *rsa.PublicKey, expectedAud, expectedIss string) error
+	VerifyJWTWithJWKS(jwtToken, jwksURL, expectedAud, expectedIss string) error
 	VerifyJWTSignature(jwtToken string, jwtPublicKey *rsa.PublicKey) error
```

Comment:
Do we still have a usage in this older methods? 
Are we planning to use the new VerifyJWT method in all the places?

Reply:
One perfect example is Google id token. It doesn't contain the `nbf` claim.

https://developers.google.com/identity/openid-connect/openid-connect#:~:text=Google%20ID%20Tokens%20may%20contain%20the%20following%20fields